apis/swagger.yml

```diff
@@ -1169,7 +1163,20 @@ definitions:
       supernodeIP:
         type: "string"
         description: "IP address of supernode which the peer connects to"
- 
+      dfdaemon:
```

Comment:
I suggest that we could remove this parameter, this is so specific for dfdaemon. While I think we can make it more general for dfget, since we can set callSystem with value of `daemon`.

Reply:
I think these are two concepts. `dfget --dfdaemon --callsystem "cluster1"` represents a dfget request from dfdaemon, and the dfdaemon is in cluster1. It's improper to move `dfdaemon` to `callsystem` since they have different meaning. 

But I am wondering do we have any plan about `--dfdaemon` later? Will there be any difference between pure dfget and dfget from dfdaemon? I think black/white list will only apply to `callSystem`, if that, removing this parameter makes sense.